app/routes/base.typography.tsx

```diff
@@ -17,13 +17,16 @@ export default function Page() {
 	return (
 		<div>
 			<h1 className="text-5xl font-medium">Typography</h1>
-			<p className="mt-4 text-xl text-gray-600">Very early explorations of typographic scale.</p>
+			<p className="mt-4 text-xl text-default">Very early explorations of typographic scale.</p>
 			<h1 className="mt-8 text-5xl font-medium">Heading 1</h1>
 			<h2 className="mt-4 text-3xl font-medium">Heading 2</h2>
 			<h3 className="mt-4 text-2xl font-medium">Heading 3</h3>
 			<h4 className="mt-4 text-xl font-medium">Heading 4</h4>
 			<h5 className="mt-4 text-base font-medium">Heading 5</h5>
 			<h6 className="mt-4 text-xs font-medium uppercase tracking-widest">Heading 6</h6>
+
+			<div className="text-strong">Primary text</div>
+			<div className="text-default">Secondary text</div>
```

Comment:
Don't forget to include the other text styles here for quick reference.

Reply:
thanks! adding now